tools/MINGW-packages/mingw-w64-opencolorio2-git/PKGBUILD

```diff
@@ -66,7 +68,6 @@ build() {
 package() {
   cd build-${MINGW_CHOST}
   make DESTDIR=${pkgdir} install
-  rm -r ${pkgdir}/msys64*
```

Comment:
This line was trying to remove directories for pystring and openexr stuff. Adding these libs to the depends section above prevents these directories from being created.

The rm command is also assuming that msys2 is installed in the root directory of the drive and fails if msys2 was installed elsewhere (e.g. c:\tools\msys64)